ui/components/issuesidebar/issuesidebar.go

```diff
@@ -71,9 +73,12 @@ func (m Model) Update(msg tea.Msg) (Model, tea.Cmd) {
 				return m, cmd
 
 			case tea.KeyEsc, tea.KeyCtrlC:
-				m.inputBox.Blur()
-				m.isCommenting = false
-				return m, nil
+				if m.shouldCancelComment() {
+					return m, nil
+				}
+			default:
+				m.inputBox.SetPrompt(commentPrompt)
+				m.ShowConfirmCancel = false
 			}
 
 			m.inputBox, taCmd = m.inputBox.Update(msg)
```

Comment:
make sure you don't pass the confirmation key to the input here.
currently, if you press `n`, it cancels but it also inserts an `n` into the input.